custom_model_runner/datarobot_drum/resource/components/Python/prediction_server/prediction_server.py

```diff
@@ -152,6 +152,9 @@ def info():
 
         @model_api.route("/health/", methods=["GET"])
         def health():
+            if hasattr(self._predictor, "health_check"):
+                return self._predictor.health_check()
```

Comment:
@elatt this should do the trick with the liveness check.

Reply:
@elatt 
The `custom_model_wait_for_server` uses the `/ping` or `/` root route for the liveliness check:
https://github.com/datarobot/datarobot-user-models/blob/master/custom_model_runner/datarobot_drum/drum/server.py#L28-L34

The `/health` route is supposed to be used for a readiness check
https://github.com/datarobot/datarobot-user-models/blob/master/custom_model_runner/README.md?plain=1#L196-L209

But as I see, it's not invoked by anything. If model is not loaded / or NIM is not started, the `/ping` route is always green.

Reply:
I think `custom_model_wait_for_server` uses `/health`

Reply:
For custom models the root path '/' is being used. (via `custom_model_wait_for_server`).
I think it would be a good enhancement to apply the same functionality also for the `/` and `/ping` endpoints, by conditionality delegating the call to the predictor (if the method is implemented there).

Reply:
@zohar-mizrahi thank you, I'll add the change